x-pack/plugins/enterprise_search/public/applications/__mocks__/kibana_context.mock.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { httpServiceMock } from 'src/core/public/mocks';
+
+/**
+ * A set of default Kibana context values to use across component tests.
+ * @see enterprise_search/public/index.tsx for the KibanaContext definition/import
+ */
+export const mockKibanaContext = {
+  http: httpServiceMock.createSetupContract(),
+  setBreadcrumbs: jest.fn(),
+  enterpriseSearchUrl: 'http://localhost:3002',
```

Comment:
Should we DRY out this url somewhere in case we buy Google or something and they have that port used and we need to change it in the future?

Reply:
For test mocks? No IMO, since it doesn't really matter what the URL is - it's essentially just a placeholder.

This is also not a hard and fast rule thing, but I'm generally against over-refactoring tests with variables when it comes to static strings. Just IMO, test specs should enable developers to understand what the code is attempting to accomplish, and it's much easier to quickly skim `http://localhost:3002/some/url/path` and understand what the generated URL should look like, instead of `${hostUrl}/${somePath}`